sdk/src/test/java/io/dapr/it/state/HelloWorldClientIT.java

```diff
@@ -0,0 +1,86 @@
+package io.dapr.it.state;
+
+import io.dapr.DaprGrpc;
+import io.dapr.DaprProtos;
+import io.dapr.it.DaprIntegrationTestingRunner;
+import io.dapr.it.services.HelloWorldGrpcStateService;
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+import org.junit.*;
+
+
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeoutException;
+
+
+public class HelloWorldClientIT {
+
+
+    private static DaprIntegrationTestingRunner daprIntegrationTestingRunner;
+
+    @BeforeClass
+    public static void init() throws IOException, InterruptedException, TimeoutException, ExecutionException {
```

Comment:
Just throws Exception for simplicity. It is OK for tests.

Reply:
yes, the idea is to fail with an exception the complete build

Reply:
My comment was just to do the following:
public static void init() throws Exception {

Instead of
public static void init() throws IOException, InterruptedException, TimeoutException, ExecutionException {

Reply:
ok, I will

Reply:
this is done